.github/workflows/update-metadata-glsa.yml

```diff
@@ -0,0 +1,58 @@
+name: Keep GLSA metadata updated
+on:
+  schedule:
+    - cron:  '0 7 1 * 1'
```

Comment:
This means do perform the action [at 7 in the morning on the first day of a month and on Monday](https://crontab.guru/#0_7_1_*_1). I have no idea if this means that this will be executed only when first day of a month is Monday at the same time, or it will be executed twice when first day of a month is not Monday (once on the first day, second time on Monday). Or scheduler will be smart and execute it on the first Monday after first day of a month…

Reply:
fixed the crontab